stories/titleAndListWithOptionalLogos.stories.js

```diff
@@ -4,16 +4,23 @@ import Theme from './theme'
 import BlueBackground from '../src/components/Common/BlueBackground'
 import GreyBackground from '../src/components/Common/GreyBackground'
 import { Grid } from '../src/components/grid'
-import TitleAndList from '../src/components/Common/TitleAndList'
+import TitleAndList from '../src/components/Common/TitleAndListWithOptionalLogos'
 import StyledLink from '../src/components/Common/StyledLink'
 
 addDecorator(Theme)
 
+const ExampleImage = {
+  title: 'Community Support Logo',
+  file: { url: '/community_support.svg' },
+  className: 'logo'
+}
+
```

Comment:
This is now used twice ☝️  Maybe worth making an export import in one of the files.
Same with body, or any re-used filed.

Reply:
I don't think we will need the WhyOpenSource story, once the section is on the open source page. So it's prob fine to leave the duplication for now?